game/monster.py

```diff
@@ -89,7 +89,13 @@ def update_weapon(self, weapon):
 
         # Update weapon transform
         weaponxf = self.form.weapon_offset
+        def __scale_down__(array):
+            array[0] = array[0] * .1
+            array[1] = array[1] * .1
+            array[2] = array[2] * .1
+            return array
         pos, hpr, scale = weaponxf['position'][:], weaponxf['hpr'][:], weaponxf['scale'][:]
+        pos = __scale_down__(pos)
```

Comment:
This could simply be:
```py
pos = [i * 0.1 for i in pos]
```

Although, should we be doing this fix here, or should be be adjusting the values coming in from the weapon_offset?